packages/console/src/App.tsx

```diff
@@ -4,6 +4,8 @@ import React, { useEffect } from 'react';
 import { BrowserRouter, Navigate, Route, Routes, useLocation, useNavigate } from 'react-router-dom';
 import useSWR, { SWRConfig } from 'swr';
 import './scss/normalized.scss';
+// eslint-disable-next-line import/no-unassigned-import
+import '@fontsource/roboto-mono';
```

Comment:
what's the size of this package? should we use `React.lazy`?

Reply:
Before:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/12833674/168427447-8e276645-fd9a-4798-9350-664e8cf85d58.png">
<img width="492" alt="image" src="https://user-images.githubusercontent.com/12833674/168427516-417a8bbc-7c40-44ed-8b56-f42d80c3a204.png">


After:
<img width="582" alt="image" src="https://user-images.githubusercontent.com/12833674/168427691-3d895ac0-0c2b-4165-b06e-16e3602e0ce3.png">
<img width="580" alt="image" src="https://user-images.githubusercontent.com/12833674/168427722-9362439c-9380-451b-a4f3-428046cebb0f.png">

I tried build the project before and after this PR change, and the result is above.
From the screenshot, we can tell the sizes of the main js and css files are barely changed, and the font faces (.woff2) are built separately.

With that being said, I think we are fine. And self-hosting the fonts is way better than loading them through CSS in runtime.